src/main/asciidoc/index.adoc

```diff
@@ -89,12 +89,56 @@ Publish a message to a queue and confirm the broker acknowledged it.
 
 === Consume
 
-Consume messages from a queue
+Consume messages from a queue.
 
 [source,$lang]
 ----
-// Create the event bus handler which messages will be sent to
-{@link examples.RabbitMQExamples#basicConsume}
+// Create a stream of messages from a queue
+{@link examples.RabbitMQExamples#basicConsumer(io.vertx.core.Vertx, io.vertx.rabbitmq.RabbitMQClient)}
+----
+
+In any moment of time you can pause or resume the stream. So when stream is paused it is guarantied that you will not receive any messages.
+
+[source,$lang]
+----
+{@link examples.RabbitMQExamples#pauseAndResumeConsumer(io.vertx.rabbitmq.RabbitMQConsumer)}
+----
+
+There are actually a set options to specify when creating a consumption stream.
```

Comment:
this sentence is missing what you can do with these options

Reply:
just added explanation of each option